Core1Platform/UsingCore1/01a_UserSetupActindo.md

```diff
@@ -0,0 +1,74 @@
+[!!Set up U2F from the administrator side](../AdministratingCore1/03_AdminSetupActindo.md)
+[!!User interface Profile settings](../UserInterface/01d_U2F.md)
+
+# Set up U2F from the user side
+
+The secure access to Actindo can be granted by using *Multi-Factor Authentication (MFA)*. The *MFA authentication* method requires two or more verification factors to log in to an application or website. Actindo uses the *YubiKey* as a second factor for the two-factor authentication, using the *Universal Second Factor (U2F)* standard.
+
+For the U2F setup of an Actindo account, a user-side setup as well as an administrator-side setup is necessary. As a user, you need to define several settings within your Actindo account to add *U2F* with the *YubiKey* as an authorization method.
+
+#### Prerequisites
+
+- You possess an *YubiKey*.
+- You have an active Actindo account.
+
+#### Procedure
+
+*Login Actindo*
+
+![Login username](../../Assets/Screenshots/Core1Platform/LoginUserName.png "[Login username]")
+
+> [Info] For the user-side setup, you can log in to both a live account and a sandbox of the Core1 Platform.
+
+1. Enter your username in the *Username* field and click the [LOGIN] button.   
+    The *Password* field is displayed.
+
+    ![Login password](../../Assets/Screenshots/Core1Platform/LoginPassword.png "[Login password]")
+
+2. Enter your password in the *Password* field and click the [LOGIN] button.   
+    You are logged in. The *Actindo Dashboard* is displayed.
+
+    ![Actindo dashboard](../../Assets/Screenshots/ActindoDashboard/ActindoDashboard.png "[Actindo dashboard]")
+
+3. Click your avatar icon in the upper right corner.   
+    The profile menu is displayed.
+
+    ![Profile menu](../../Assets/Screenshots/Core1Platform/ProfileMenu.png "[Profile menu]")
+
+4. Click the *Settings* entry in the profile menu.   
+    The *Profile* tab is displayed.
+
+    ![Profile](../../Assets/Screenshots/Core1Platform/ProfileSettings/Profile/Profile.png "[Profile]")
+
+5. Click to the *U2F* tab.   
+    The *U2F* tab is displayed.
+
+    ![U2F](../../Assets/Screenshots/Core1Platform/ProfileSettings/U2F/U2F.png "[U2F]")
+
+6. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Add new U2F device* window is displayed.
+
+    ![Add new U2F device](../../Assets/Screenshots/Core1Platform/ProfileSettings/U2F/AddNewU2FDevice.png "[Add new U2F device]")
+
+7. Click the [OK] button to continue the device registration for U2F.   
+    The browser window *Verify your identity* is displayed to select the device type for the identity verification.
+
```

Comment:
Also the pictures shown, can differ from key to key. Some keys also need an Windows Pin that you can chose yourself. 
So inbetween there will be a prompt to chose a windows pin before you can touch the U2F Key.